hive/runners/single_agent_loop.py

```diff
@@ -153,7 +153,6 @@ def set_up_experiment(config):
     agent = agent_fn(
         observation_space=env_spec.observation_space[0],
         action_space=env_spec.action_space[0],
-        stack_size=config.get("stack_size", 1),
```

Comment:
This shouldn't be removed. It will break other agents.

Reply:
done